scripts/pdfsheet.js

```diff
@@ -32,6 +32,22 @@ Hooks.on("init", () => {
 			},
 		});
 	}
+
+	window.translate = function(sourceText, sourceLang, targetLang){
+		$.ajaxSetup({
+			async: false
+		});
+		var retString = "";
+		if(sourceText) {
+			console.log(sourceText);
+			var url = "https://translate.googleapis.com/translate_a/single?client=gtx&sl="+ sourceLang + "&tl=" + targetLang + "&dt=t&q=" + encodeURI(sourceText);
+			$.getJSON(url, function(data) {
+				retString = data[0][0][0];
+			});
+			console.log(retString);
+		}
+		return retString;
+	}
```

Comment:
I don't think it makes sense for this module to be adding to the window object. The translation function should be added within the scope of the module's API.

Reply:
`game.modules.get("pdf-sheet").api.translate = ...`

see https://foundryvtt.wiki/en/development/guides/package-best-practices#module-apis-and-dependencies